### ISSUE TITLE
Verifying webhooks

### ISSUE THREAD

**Issue by aaronn** [replicate/replicate-python#312]
It looks like only the [javascript](https://github.com/replicate/replicate-javascript#verifying-webhooks)and [golang](https://github.com/replicate/replicate-go#webhooks) version of the client libraries have webhook validation right now. Would be great if this came to the Python library.

**Comment by mattt** [comment]
Hi @aaronn. That's a great suggestion! I just added this with #321 and #322, which are now available in [0.28.0](https://github.com/replicate/replicate-python/releases/tag/0.28.0).

```python
import replicate
secret = replicate.webhooks.default.secret()

replicate.webhooks.validate(request=request, secret=secret)
```

**Comment by maximilianvb** [comment]
Hi, if anyone else confused about how to use this with python fastapi here is some code that works for me:

```
@app.post("/replicate-webhook")
async def replicate_webhook(request: Request, task_id: str = Query(...)):
    body = await request.body()
    body = body.decode()
    # Get the headers
    headers = dict(request.headers)
    # Validate the webhook
    secret = replicate.webhooks.default.secret()

    replicate.webhooks.validate(body=body, headers=headers, secret=secret)
   ```
